.github/workflows/main.yml

```diff
@@ -8,6 +8,9 @@ jobs:
         uses: actions/checkout@v2
       - name: Set up Python
         uses: actions/setup-python@v2
+      - name: Install nmigen
+        run: |
+          pip install git+https://github.com/nmigen/nmigen
```

Comment:
Please drop this commit. I'm aware the CI is red, but the problem goes beyond CI to everyone who uses nmigen-boards, so it is a genuine issue with the repo rather than just a CI omission.

Reply:
I think you dropped the wrong one

Reply:
Oh, that commit wasn't supposed to be here. No clue how that happened, sorry.